dev/AppNotifications/AppNotificationBuilder/AppNotificationBuilder.h

```diff
@@ -59,13 +64,22 @@ namespace winrt::Microsoft::Windows::AppNotifications::Builder::implementation
         static bool IsUrgentScenarioSupported();
 
     private:
+        void ThrowIfMaxInputItemsExceeded();
         std::wstring GetDuration();
         std::wstring GetScenario();
         std::wstring GetArguments();
         std::wstring GetButtonStyle();
         std::wstring GetText();
         std::wstring GetImages();
-        std::wstring GetButtons();
+        std::wstring GetActions();
+
+        struct TextBox
```

Comment:
I think we can remove this if we construct the element inside the AddTextBox(...) functions.

Reply:
I think you're right. That would be useful to do extra validation at build time (for example, to ensure that the input hint on a button, actually match a textbox), but that something we can re-add later, if we ever revisit the builder.

Reply:
done